CMakeLists.txt

```diff
@@ -0,0 +1,99 @@
+cmake_minimum_required(VERSION 3.15)
+cmake_policy(SET CMP0091 NEW) # Use MSVC_RUNTIME_LIBRARY to select the runtime
+project(Verilator
+    VERSION 4.025.99
```

Comment:
Still not sure how we are maintaining this

Reply:
currently it will be manual editing on version change